### PR TITLE
Apiman OSGI UI bundle

### PR DESCRIPTION
**Pull request by cmoulliard**
- Refactor the project to create the Apiman OSGI UI bundle
- Add the option "attachclasses" option to the maven war plugin of Apiman War ui in order to generate the jar containing the code of the project
- Expose IUIConfig as OSGI Service
- Include an Activator to register the different servlets
- Add missing packages

### CODE REVIEW

File: manager/ui/war/apiman/config.js-SAMPLE [hunk 1]
```diff
@@ -12,7 +12,7 @@ var APIMAN_CONFIG_DATA = {
     "metrics" : true
   },
   "api" : {
-    "endpoint" : "http://localhost:8080/apiman",
+    "endpoint" : "http://localhost:8181/apimanui/apiman",
```
Comment (EricWittmann) [inline manager/ui/war/apiman/config.js-SAMPLE]: Please roll this change back - the sample config should be 8080.  Also not sure why it would ever be /apimanui/apiman?

File: manager/ui/war/src/main/java/io/apiman/manager/ui/server/impl/UIConfig.java [hunk 2]
```diff
@@ -13,9 +13,10 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.apiman.manager.ui.server;
+package io.apiman.manager.ui.server.impl;
```
Comment (EricWittmann) [inline manager/ui/war/src/main/java/io/apiman/manager/ui/server/impl/UIConfig.java]: Why this change?

Comment (cmoulliard) [inline manager/ui/war/src/main/java/io/apiman/manager/ui/server/impl/UIConfig.java]: To avoid that the classloader of apiman-common-util imports the impl class located here previsouly "io.apiman.manager.ui.server" as we will use another class for the osgi bundle of the apiman manager ui

File: manager/ui/war/apiman/config.js-SAMPLE (hunk 1 above)
Comment (cmoulliard) [inline manager/ui/war/apiman/config.js-SAMPLE]: ok. will do that
